HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -170,10 +170,27 @@ def customiseFor2017DtUnpacking(process):
 
     return process
 
+def customiseFor31295(process):
```

Comment:
I understand this fragment will be called in production to update the linker parameters with the right values. I wonder if there is a more robust way of propagating the pfblock configuration updates to HLT, as here we are overriding the types, and also relying quite heavily on the particular PSet structure.

I think also the function name could perhaps be made a bit more descriptive.

Reply:
I think it's a normal convention to use this format, but I added a more description to this python file.